.github/workflows/build.yml

```diff
@@ -72,6 +72,9 @@ jobs:
       - name: Run lint
         run: yarn run lint
 
+      - name: Check for circular dependencies
+        run: npx madge --exclude '(bitstream|bundle|collection|config-submission-form|eperson|item|version)\.model\.ts$' --circular --extensions ts ./
```

Comment:
I think it could be useful if this command can be launched also locally, could you please add shortcut in the list of scripts to our package.json?

Reply:
That's a great addition! Added it to the package.json configuration. Could you please review this again?